NetworkService/Tests/NetworkServiceTests/NetworkServiceTests.swift

```diff
@@ -0,0 +1,8 @@
+import XCTest
+@testable import NetworkService
```

Comment:
Could you please explain why do we need a `testable` import here?

Reply:
To allow access to the internal entities of the NetworkService module in NetworkServiceTests.

Reply:
What internal entities you want to expose from the `NetworkService`?

Reply:
init() {}

Reply:
Ok, then why the `init()` has the internal scope?
Shouldn't that be public?

Reply:
i created a public `init()`